content/docs/2.10/deploy.md

```diff
@@ -81,16 +81,22 @@ Locate installed KEDA Operator in `keda` namespace, then remove created `KedaCon
 
 If you want to try KEDA on [Minikube](https://minikube.sigs.k8s.io) or a different Kubernetes deployment without using Helm you can still deploy it with `kubectl`.
 
-- We provide sample YAML declaration which includes our CRDs and all other resources in a file which is available on the [GitHub releases](https://github.com/kedacore/keda/releases) page.
+- We provide sample YAML declaration which includes our CRDs and all other resources in a file which is available on the [GitHub releases](https://github.com/kedacore/keda/releases) page. There are 2 files available, with and without the admission webhooks (keda-2.xx.x.yaml and keda-2.xx.x-core.yaml respectively), you can chose the installation mode just using the correct file.
```

Comment:
This is a bit long
![image](https://user-images.githubusercontent.com/4345663/210948835-a954c28b-eee3-4264-8ca9-cb9a617e5190.png)

Let's simplify
```suggestion
- We provide sample YAML declaration which includes our CRDs and all other resources in a file which is available on the [GitHub releases](https://github.com/kedacore/keda/releases) page.
  - We offer two options to deploy KEDA:
    - Use `keda-2.xx.x.yaml` that includes all features, including admission webhooks (recommended)
    - Use `keda-2.xx.x-core.yaml` that installs the minimal required KEDA components, without admission webhooks
```

What do you think?

Reply:
nice!